### DISCUSSION TITLE
What is the differences between `new ClassName` and `new &ClassName`?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2944] [Q&A] [answered]
On https://github.com/pascalabcnet/pascalabcnet/discussions/2931, `var lbl1 := new Label;` will not compile but `var lbl1 := new &Label;` will compile. Why?

**Comment by SunSerega** [comment, marked as answer]
Because `label` is a keyword, used to define `goto` landing spots. And `&` is an escape symbol.
